gcloud/storage/blob.py

```diff
@@ -150,9 +150,9 @@ def public_url(self):
             quoted_name=quote(self.name, safe=''))
 
     def generate_signed_url(self, expiration, method='GET',
-                            client=None, credentials=None,
-                            response_type=None, response_disposition=None,
-                            generation=None):
+                            content_type='application/octet-stream',
```

Comment:
RE: `application/octet-stream` vs. `None`. Why does it prevent undefined behavior? The method being called defines what happens when `None` is passed in.